### ISSUE TITLE
Update extra_requirements: Error when running `transformers` test locally

### ISSUE THREAD

**Issue by JoanFM** [jina-ai/jina#753]
**Describe your problem**
When running locally: 

`pytest tests/unit/executors/encoders/nlp/test_mock_transformer.py`

I tried this `test` after observing this same behavior running `urbandict` example.

I receive `FAILED` results:

```bash
FAILED tests/unit/executors/encoders/nlp/test_mock_transformer.py::TransformerEncoderWithMockedModelTestCase::test_encodes_bert_like - ValueError: expected sequence of length 6 at dim 1 (got 5)
FAILED tests/unit/executors/encoders/nlp/test_mock_transformer.py::TransformerEncoderWithMockedModelTestCase::test_encodes_lm_like - ValueError: expected sequence of length 4 at dim 1 (got 3)
FAILED tests/unit/executors/encoders/nlp/test_mock_transformer.py::TransformerEncoderWithMockedModelTestCase::test_loads_tf_encoder - ValueError: Can't convert non-rectangular Python sequence to Tensor.
```

**Possible reason**
Is there any guarantee that batch_encode_plus will return input_ids  of the same size for a given batch? Even when there are less tokens than `max_length`?

**Environment**

```bash
platform linux -- Python 3.7.5, pytest-5.4.2, py-1.8.1, pluggy-0.13.1 -- /usr/bin/python3.7

Name: transformers
Version: 2.9.0
Summary: State-of-the-art Natural Language Processing for TensorFlow 2.0 and PyTorch
Home-page: https://github.com/huggingface/transformers
Author: Thomas Wolf, Lysandre Debut, Victor Sanh, Julien Chaumond, Sam Shleifer, Google AI Language Team Authors, Open AI team Authors, Facebook AI Authors, Carnegie Mellon University Authors
Author-email: thomas@huggingface.co
License: Apache
Location: LOCATION
Requires: regex, filelock, tokenizers, sentencepiece, tqdm, numpy, sacremoses, requests
Required-by: flair

Name: torch
Version: 1.5.0
Summary: Tensors and Dynamic neural networks in Python with strong GPU acceleration
Home-page: https://pytorch.org/
Author: PyTorch Team
Author-email: packages@pytorch.org
License: BSD-3
Location: LOCATION
Requires: numpy, future
Required-by: torchvision, laserembeddings, flair

Name: tensorflow
Version: 2.0.0
Summary: TensorFlow is an open source machine learning framework for everyone.
Home-page: https://www.tensorflow.org/
Author: Google Inc.
Author-email: packages@tensorflow.org
License: Apache 2.0
Location: LOCATION
Requires: numpy, wrapt, absl-py, wheel, google-pasta, opt-einsum, keras-preprocessing, protobuf, termcolor, tensorflow-estimator, astor, six, tensorboard, grpcio, gast, keras-applications
Required-by: 
```

┆Issue is synchronized with this [Jira Task](https://jinaai.atlassian.net/browse/JINACORE-138) by [Unito](https://www.unito.io/learn-more)

**Comment by JoanFM** [comment]
This is solved when uploading to `transformers==3.0.2` and `tensorflow==2.3.0`

**Comment by JoanFM** [comment]
#756 make this change quite likely unnecessary
